examples/spi/host/src/host.c

```diff
@@ -31,27 +37,36 @@ int main(void)
   }
 
   printf("started\n");
+  printf("using payload of size %d\n", PAYLOAD_LEN);
```

Comment:
not very flexible if its hardcoded? or am I missing something

Reply:
Is configurable at compile time, before was hardcoded to be 1. We do not expect runtime variable length because it can overcomplicate the example.

Reply:
Perhaps some extra docs to make it clear whats going on, otherwise this work doesn't add much value. The reader will need to know that this feature exists in order for it to be of any use

Reply:
The benefit is that to test a transfer of Xbytes for example (for instance to compare timings with the one on fwk_rtos) is much easier to have it as a define here.